### PR TITLE
update rsyslog rules

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

provided instructions how to setup remote name and CA
This info was added to rule descriptions and ocil clauses of rsyslog_remote_tls and rsyslog_remote_tls_cacert

#### Rationale:

the description and ocil was not entirely clearAs

### CODE REVIEW

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ description: |-
     <pre>action(type="omfwd" protocol="tcp" Target="&lt;remote system>" port="6514"
         StreamDriver="gtls" StreamDriverMode="1" StreamDriverAuthMode="x509/name")
     </pre>
+    Replace the <tt>&lt;remote system></tt> in the above configuration with an IP address or a host name of the remote logging server.
```
Comment (yuumasato) [inline linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml]: Could you also add the command to achieve compliance?
A command that echoes the settings into the config file, probably above this line. As the "Replace the <tt>&lt;remote system></tt>  " text goes well with it.

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls_cacert/rule.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ description: |-
     using correct path for the <tt>DefaultNetstreamDriverCAFile</tt>
     global option in <tt>/etc/rsyslog.conf</tt>, for example
     <pre>global(DefaultNetstreamDriverCAFile="/etc/pki/tls/cert.pem") *.*</pre>
+    Replace the <tt>/etc/pki/tls/cert.pem</tt> in the configuration line above with the path to the file with CA certificate generated for the purpose of remote logging.
```
Comment (yuumasato) [inline linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls_cacert/rule.yml]: Same as in the other rule, please just add a command to configure the rule.

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml [hunk 3]
```diff
@@ -12,6 +12,8 @@ description: |-
...
+
```
Comment (yuumasato) [inline linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml]: Extra unnecessary blank line.

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls_cacert/rule.yml (hunk 2 above)
Comment (vojtapolasek) [reply linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls_cacert/rule.yml]: Wouldn't be the exact command more suitable for ocil?

Comment (yuumasato) [reply linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls_cacert/rule.yml]: In the OCIL the command is for checking, here is to configure.

**Comment by yuumasato** [comment]
LGTM.
@openscap-ci test this please

**Comment by yuumasato** [comment]
There was some technical issue on Fedora build.
@openscap-ci test this please

**Pull request merged**